### PR TITLE
replace usage of warnings for deprecations

### PR DESCRIPTION
**Pull request by MartinNowak**

### CODE REVIEW

File: deprecate.dd [hunk 1]
```diff
@@ -9,47 +9,45 @@ $(SPEC_S Deprecated Features,
...
     )
 
     $(DL
         $(DT Spec)
         $(DD Removal from the Specification)
-        $(DT Warn)
-        $(DD The compiler emits a warning if the feature is used)
         $(DT Dep)
-        $(DD The compiler issues an error if not compiled with the -d switch)
+        $(DD The compiler warns by default, issues an error with the -de switch, and can be silenced with the -d switch)
```
Comment (mathias-lang-sociomantic) [inline deprecate.dd]: Maybe we could use another adjective than "warns" so it doesn't get confused with warnings. I would just say "The compiler emits a 'Deprecation' message on usage by default, ...".

A bit OT:
I've always found DMD usage (`  -dw            show use of deprecated features as warnings (default)`) confusing, as I expect `dmd -dw -w` to have the same effect as `dmd -de` for deprecations.

**Comment by mathias-lang-sociomantic** [comment]
Besides the wording suggestion, LGTM.

**Comment by andralex** [comment]
guess I'll pull this, feel free to adjust the wording later

**Comment by CyberShadow** [comment]
Wasn't this blocked by the dmd pull?

File: deprecate.dd (hunk 1 above)
Comment (MartinNowak) [inline deprecate.dd]: Not sure what the use-case for `-dw` is. If you have a better wording, just make a PR.

**Pull request merged**